db/seeds/development_users.rb

```diff
@@ -3,7 +3,7 @@
   email: "roda@dxw.com"
 )
 partner_org_user = User.find_or_initialize_by(
-  name: "artner Org User",
+  name: "Partner Org User",
```

Comment:
Hah, this has been bugging me for way too long. Thanks for doing something about it